Project.toml

```
@@ -15,10 +15,3 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 Crayons = "4.0"
 StatsBase = "0.32, 0.33"
 julia = "1"
```

I see you have moved the test targets to `test/Project.toml` and that makes this package not compatible with old Julia versions. Given that it's about to make Julia 1.6 the next LTS release, I'm okay with this change.

```suggestion
julia = "1.2"
```

---

For compatibility consideration, I personally prefer to still manage `[targets.test]` entries and not bump `julia` compat until we really need some new features that can't be easily patched. For instance, the `test/Project.toml` and something like `isnothing` can be perfectly written in another way and thus aren't good reasons to bump the `julia` compat.

Just my two cents.

---

I'd prefer also to run the CI under the current LTS which is `1.0.5`.